recipes-devtools/android-simg2img/android-simg2img_git.bb

```diff
@@ -0,0 +1,21 @@
+SUMMARY = "Tool to convert Android sparse images"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://img2simg.cpp;beginline=1;endline=15;md5=69dd3a3cbb50842da4c61d01ee32f421"
+
+SRC_URI = "git://github.com/MagneFire/android-simg2img.git;protocol=https"
```

Comment:
I'd rather have this hosted under the AsteroidOS github organization, for better visibility